packages/storybook-vue/src/stories/CustomRadioButton/CustomRadioButtonWithFieldsetAndLegend.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { action } from '@storybook/addon-actions';
+import type { Meta, StoryFn } from '@storybook/vue3';
+import { CustomRadioButton } from '@utrecht/component-library-vue';
+import { argTypes, data } from './argTypes';
+export default {
+  component: CustomRadioButton,
+  id: 'vue-custom-radio-button-with-field-set-and-legend',
+  title: 'Vue.js Component/Custom Radio Button/ Custom Radio Button With Fieldset And Legend',
+  argTypes: argTypes,
+} as Meta<typeof CustomRadioButton>;
+
+const Template: StoryFn<typeof CustomRadioButton & { radioButtons: typeof data }> = (args) => ({
+  components: { CustomRadioButton },
+  setup() {
+    return { args, options: args.radioButtons };
+  },
+  template: `
+  <form>
+  <FormFieldset>
+  <FormFieldsetLegend>Kies locatie</FormFieldsetLegend>
+  <FormField v-for="{id, label, name} in options" :key="id">
+    <FormLabel :for="id" type="radio" :checked="args.checked" :disabled="args.disabled">
+        <CustomRadioButton :name="name" v-bind="args" :id="id" :value="id" @update:nodeValue="action"/>
```

Comment:
Is `nodeValue` correct here, or should it be `modelValue`?

Reply:
`modelValue` the correct one I will update it